linfa-kernel/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "linfa-kernel"
+version = "0.1.0"
+authors = ["Lorenz Schmidt <bytesnake@mailbox.org>"]
+description = "Kernel methods for non-linear algorithms"
+edition = "2018"
+license = "MIT/Apache-2.0"
+
+repository = "https://github.com/LukeMathWalker/linfa"
```

Comment:
Should this be `repository = "https://github.com/rust-ml/linfa"`?

Reply:
yes that's out of date